docs/configuration/authentication/oidc_digid.rst

```diff
@@ -57,28 +77,47 @@ Zorg dat je de volgende :ref:`gegevens <configuration_oidc_digid_appgroup>` hebt
 * Client ID
 * Client secret
 
-Navigeer vervolgens in de admin naar **Configuratie** > **OpenID Connect configuration for DigiD**.
+Navigeer vervolgens in de admin naar **Configuratie** > **DigiD (OIDC)**.
 
-1. Vink *Enable* aan om OIDC in te schakelen.
-2. Laat bij **BSN claim name** de standaardwaarde staan, tenzij de naam van het BSN veld
-   in de OIDC claims anders is dan ``bsn``.
-3. Vul bij **OpenID Connect client ID** het Client ID in, bijvoorbeeld
+Stel de algemene instellingen in:
+
+1. Vink *Ingeschakeld* aan om OIDC in te schakelen.
+2. Vul bij **OpenID Connect client ID** het Client ID in, bijvoorbeeld
    ``a7d14516-8b20-418f-b34e-25f53c930948``.
-4. Vul bij **OpenID Connect secret** het Client secret in, bijvoobeeld
+3. Vul bij **OpenID Connect secret** het Client secret in, bijvoobeeld
    ``97d663a9-3624-4930-90c7-2b90635bd990``.
-5. Laat bij **OpenID Connect scopes** de standaardwaarden staan.
-6. Vul bij **OpenID sign algorithm** ``RS256`` in.
-7. Laat **Sign key** leeg.
+4. Laat bij **OpenID Connect scopes** de standaardwaarden staan, of stel deze in volgens
+   de instructies van je OpenID Connect provider.
+5. Vul bij **OpenID sign algorithm** ``RS256`` in.
+6. Laat **Sign key** leeg.
 
-Vervolgens moeten er een aantal endpoints van de OIDC provider ingesteld worden,
-deze kunnen automatisch bepaald worden aan de hand van het discovery endpoint
+Stel dan de claims in:
+
+7. Laat bij **BSN-claim** de standaardwaarde staan, tenzij de naam van het BSN veld
```

Comment:
Yeah I've mixed feelings about this. I think we can get away with it for now because it's mostly us or technical people configuring this that self-host, and a quick check/trial and error should be enough if they can't figure it out?

I do feel there's a lack of "general" OIDC documentation - but perhaps we'll be able to tackle that with improvements in the library (e.g. that issue to configure OIDC provider endpoints once and re-use them in different configs/clients), and we might be able to merge more concepts/UI and from that the documentation follows logically.